documentation/getting-started/rest/uploading-data.mdx

```diff
@@ -24,3 +25,12 @@ To upload a file, the data you send in a `POST` request must be of the content t
 3. To add your image file click in the body tab and select `multipart/form-data` in the content-type dropdown.
 4. Give your file a name and click on `choose files` to select your file.
 5. Click "**Send**" to upload your file.
+
+## Uploading Binary Data
+
+When uploading binary files, you may want to send raw binary data instead of files in a multipart form. This is typically done using the `application/octet-stream` content type. Follow these steps to upload binary data:
+
+1. Select the **POST** or **PUT** HTTP method and set your API Endpoint URL.
+2. In the **Body** tab, select `application/octet-stream` from the Content Type dropdown.
+3. Upload your binary data by selecting the file from your local machine.
+4. Click **Send** to upload the binary data.
```

Comment:
The Desktop App doesn't support this currently; might be helpful to mention a note.

Reply:
Added a note ✅